[1] - Updated Login Page

akash-kumar-dev opened the pull request:
Addresses: #1 

![image](https://github.com/user-attachments/assets/4c30561e-6305-4e13-ba81-baa5d3e818e0)

# Login Page Redesigned

Reference:   [Figma Design](https://www.figma.com/design/pn2JO3bbMEzztJHuXJnS10/nise3-skill-and-certification?node-id=0-1&t=v4yFDuE6jZqnIODw-1)

Shelly011s commented:
Please implement the figma designs precisely. Implement the navbar, there should no background for the sign up form that is making it stand apart form background, the button colour, the scroll bar should not be there in both x-axis and y-axis.

akash-kumar-dev commented:
## fixed styles 
- Navbar Sign up and Login button styled as in Figma Design
- Forrm Border and background color removed
- Form submit button color chnaged
- removed x-y axis scroll for xl devices
- left side background color changed
- changed styles for footer responsiveness

![image](https://github.com/user-attachments/assets/6f81e3d1-fd18-4f68-a85b-cda54773b23e)
